sycl/include/sycl/ext/oneapi/matrix/matrix-unified.hpp

```diff
@@ -8,6 +8,7 @@
 
 #pragma once
 
+#include "matrix-hip.hpp"
```

Comment:
Do we need condition on including this only when AMD backend is requested like we did for NVPTX?
#if defined(__SYCL_DEVICE_ONLY__) && defined(__NVPTX__)
#include "matrix-tensorcores.hpp"
#endif

Reply:
Thanks @dkhaldi , done in 048ac39b65fdd78c9fa190433b8ed8fdc2fb118e